build-rpm.sh

```diff
@@ -79,22 +79,29 @@ if [ ! -z "$SOURCE_RPM" ]; then
         fi
 elif [ ! -z "$SPEC_FILE" ]; then
         if [ -z "$SOURCES" ]; then
-                echo "You need to specify SOURCES env variable pointing to folder or sources file (only when building with SPEC_FILE)"
-                exit 1;
+                echo "WARNING: The SOURCES env variable pointing to folder or sources file was not specified. $SPEC_FILE will need to profied it's own Source."
         fi
         echo "      SPEC_FILE:     $SPEC_FILE"
         echo "      SOURCES:       $SOURCES"
         echo "      OUTPUT_FOLDER: $OUTPUT_FOLDER"
         echo "      MOCK_DEFINES:  $MOCK_DEFINES"
```

Comment:
```suggestion
        echo "      MOCK_DEFINES:  ${MOCK_DEFINES[@]}"
```

Prints all mock defines, not just the first one.